syntax = "proto3";

message GenericMsg {
    int32 type_id = 1;
    bytes content = 2;
}
